ccd-definition/CaseEventToComplexTypes/draft-CMO/reviewCaseManagementOrder.json

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ReviewCaseManagementOrder",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "caseManagementOrder",
+    "ListElementCode": "orderDoc",
+    "EventElementLabel": "Check the order",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "READONLY"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ReviewCaseManagementOrder",
+    "CaseEventID": "draftCMO",
+    "CaseFieldID": "caseManagementOrder",
+    "ListElementCode": "hearingDate",
+    "EventElementLabel": "",
+    "FieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "FieldShowCondition": "orderDoc=\"DO_NOT_SHOW\""
```

Comment:
Yeah otherwise hearingDate shows on the page, it is probably because it is included in the caseManagementOrder.json file in the same folder but this was work done by Amit and I am not confident enough in my knowledge of ccd stuff to be changing it when I don't know the reason for it being there in the first place

Reply:
👍